yaqc_cmds/hardware/delays.py

```diff
@@ -95,14 +96,17 @@ def set_position(self, destination):
         self.set_motor_position(destination_motor)
 
     def set_offset(self, offset):
+        print("delays set_offset", self.name, offset)
         # update zero
         offset_from_here = offset - self.offset.read(self.native_units)
         offset_motor = offset_from_here / (self.native_per_motor * self.factor.read())
         new_zero = self.zero_position.read(self.motor_units) + offset_motor
         self.set_zero(new_zero)
         self.offset.write(offset, self.native_units)
+        print("new offset", self.name, self.offset.read())
         # return to old position
         destination = self.hardware.destination.read(self.native_units)
+        print("new destination", self.name, destination)
```

Comment:
a few print statements here... but not gonna bother getting rid of them immediately, as they may still be helpful during the initial stages of battle testing... I expect we will have a maintenance pr to remove prints at some point